pkgs/ffigen/lib/src/config_provider/config_types.dart

```diff
@@ -432,3 +433,16 @@ class Declaration {
     required this.originalName,
   });
 }
+
+class ExternalVersions {
+  final Versions? ios;
+  final Versions? macos;
+  const ExternalVersions({this.ios, this.macos});
+}
+
+class Versions {
+  final Version? min;
+  final Version? max;
```

Comment:
Leave TODO that max isn't supported yet.

Reply:
Done.